topic6/ContulPersoanei.java

```diff
@@ -0,0 +1,53 @@
+package topic6;
+
+public class ContulPersoanei {
+
+    private String numePersoana;
+    private Integer id;
+    private Double sumaCurenta;
+
+    public String getNumePersoana() {
+        return numePersoana;
+    }
+
+    public void setNumePersoana(String numePersoana) {
+
+    this.numePersoana = numePersoana;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) throws IllegalArgumentException{
```

Comment:
markind ca aceasta metoda arunca exceptia, chiar trebuie sa o arunci, si sa faci try cu o valoare grersita iar in catch sa transmiti o valoare correcta